tests/integration/test_ses.py

```diff
@@ -239,6 +244,47 @@ def test_send_templated_email_can_retrospect(self, ses_client, create_template):
         assert requests.delete("http://localhost:4566/_aws/ses").status_code == 204
         assert requests.get("http://localhost:4566/_aws/ses").json() == {"messages": []}
 
+    def test_sent_message_counter(self, ses_client, create_template):
+        # Ensure all email send operations correctly update the sent email counter
+        email = f"user-{short_uid()}@example.com"
+        ses_client.verify_email_address(EmailAddress=email)
+
+        counter = ses_client.get_send_quota()["SentLast24Hours"]
+
+        ses_client.send_email(
+            Source=email,
+            Message=SAMPLE_SIMPLE_EMAIL,
+            Destination={
+                "ToAddresses": ["success@example.com"],
+            },
+        )
+
+        new_counter = ses_client.get_send_quota()["SentLast24Hours"]
+        assert new_counter == counter + 1
+        counter = new_counter
+
+        create_template(Template=SAMPLE_TEMPLATE)
+        ses_client.send_templated_email(
+            Source=email,
+            Template=SAMPLE_TEMPLATE["TemplateName"],
+            TemplateData='{"A key": "A value"}',
+            Destination={
+                "ToAddresses": ["success@example.com", "lorem@ipsum.co"],
+            },
+        )
+
+        new_counter = ses_client.get_send_quota()["SentLast24Hours"]
+        assert new_counter == counter + 2
+        counter = new_counter
+
+        raw_message_data = f"From: {email}\nTo: recipient@example.com\nSubject: test\n\nThis is the message body.\n\n"
+        ses_client.send_raw_email(
+            Destinations=["foo@bar.com"], RawMessage={"Data": raw_message_data}
+        )
+
+        new_counter = ses_client.get_send_quota()["SentLast24Hours"]
+        assert new_counter == counter + 2
```

Comment:
question: I am confused, wasn't the issue report that the counter should be only +1? Or am I missing something?
After checking in moto, it seems moto is adding recipients from the raw headers to the sum. However, what if the recipient in `Destinations` and in the headers is the same? It seems they would be counted twice. (this previous part was before finding out the answer in the post under). 

Also, after finding an old bug report in LocalStack and this SO post: 
https://github.com/localstack/localstack/issues/6287
https://stackoverflow.com/questions/27446357/how-to-add-cc-and-bcc-in-amazon-ses-sendrawemail

It seems that if we have `Destinations` set then we shouldn't take into account the headers. So the count should be +1 here?

Reply:
Good catch! This will need fixes both in LS and Moto and a out of scope of this issue. For now I've updated the test and added a fixme in the SES provider